framework/web/AssetManager.php

```diff
@@ -290,6 +299,7 @@ protected function loadDummyBundle($name)
     /**
      * Returns the actual URL for the specified asset.
      * The actual URL is obtained by prepending either [[baseUrl]] or [[AssetManager::baseUrl]] to the given asset path.
+     * [[scheme]] or [[AssetManager::scheme]] is used, if specified.
```

Comment:
Yes, currently it's not obvious.
I think, I've got your idea: if AssetBundle::scheme is not set, use AssetManager::defaultScheme.
It looks good for me too. Should I do this way?